src/components/CvCheckbox/__tests__/CvCheckbox.spec.js

```diff
@@ -0,0 +1,37 @@
+import { render, fireEvent } from '@testing-library/vue';
+import CvCheckbox from '../CvCheckbox';
+
+describe('CvCheckbox', () => {
+  it('CvCheckbox - default', async () => {
+    // The render method returns a collection of utilities to query your component.
+    const { emitted, getByTestId } = render(CvCheckbox, {
+      props: { value: 'checkbox-1' },
+    });
+
+    const checkbox = getByTestId('checkbox');
+
+    await fireEvent.click(checkbox);
```

Comment:
For many events UserEvent preferred to fireEvent https://testing-library.com/docs/user-event/

```
import userEvent from '@testing-library/user-event';
...
await userEvent.click(checkbox);
...
```

Reply:
done. add the `@testing-library/user-event` package and the requested peer package `@testing-library/dom`